app/views/components/_share-links.html.erb

```diff
@@ -0,0 +1,18 @@
+<% title ||= 'Share this page' %>
+
+<div class="app-c-share-links-wrapper">
+  <h2><%= title %></h2>
+  <div class="app-c-share-links" data-module="track-share-button-clicks">
+    <%= link_to share_urls[:facebook],
```

Comment:
Would you then suggest passing them in separately to the component, e.g (ignore syntax mistakes, haven't tried it yet):
`<%= render 'components/share-links', facebook_href: @content_item.share_links[facebook_href] twitter_href:.....etc %>`

instead of pulling out the links from the share links object inside the component itself?